src/System.Text.Json/experiments/Json.Net.Test/Json.Net.Tests/JsonParser.cs

```diff
@@ -0,0 +1,565 @@
+﻿using System;
+
+namespace Json.Net.Tests
+{
+    public struct JsonParser : IDisposable
```

Comment:
It essentially can be removed. I only included it into the project whenever I wanted to deep dive into perf. They are not used in JsonNetComparison. If the files are excluded from the project, then the DLL ends up being used (as expected).

using JsonReader = System.Text.Json.JsonReader;
using JsonWriter = System.Text.Json.JsonWriter;
using JsonParser = System.Text.Json.JsonParser;